assets/scss/components/_alerts.scss

```diff
@@ -24,8 +24,9 @@
 }
 
 .alert-warning {
-  background: $beige;
-  color: $black;
+  color: #664d03;
+  background-color: #fff3cd;
+  border-color: #ffecb5;
```

Comment:
What're these changes for? Why aren't you using the variables in [/assets/scss/common/_variables.scss](https://github.com/filecoin-project/lotus-docs/blob/main/assets/scss/common/_variables.scss)?